app/src/main/java/com/example/pps_tudai/activity/MainActivity.java

```diff
@@ -4,6 +4,8 @@
 
 import android.os.Bundle;
 import com.example.pps_tudai.R;
+import com.example.pps_tudai.data.entities.AppRepository;
```

Comment:
do you use these imports here? if not, please delete it

Reply:
DONE!!